tests/utils/actions/misc.ts

```diff
@@ -369,8 +379,11 @@ export async function initEmptyDatabaseWithParagraphState(
   page: Page,
   pageId?: string
 ) {
-  const ids = await page.evaluate(pageId => {
+  const ids = await page.evaluate(async pageId => {
     const { page } = window;
+    await page.waitForLoaded();
+    page.trySyncFromExistingDoc();
```

Comment:
Is this API call required for the application side? Would be better to keep this `syncExistingDoc` method process private.

Reply:
Done